src/main/java/com/sparrowwallet/sparrow/joinstr/JoinstrPool.java

```
@@ -44,12 +46,21 @@ public JoinstrPool(String relay, String pubkey, String denomination,
         this.timeout = new SimpleStringProperty(timeout);
         this.privateKey = privateKey;
         this.status = new SimpleStringProperty("");
+
     }
 
     public String getRelay() { return relay.get(); }
     public String getPubkey() { return pubkey.get(); }
     public String getDenomination() { return denomination.get(); }
     public String getPeers() { return peers.get(); }
+    public String getPeersStatus() {
+        String connectedPeers = "0";
+        if(this.handler != null) {
+            connectedPeers = String.valueOf(handler.getConnectedPeers());
+        }
+        return connectedPeers + "/" + peers.get();
+    }
+
     public String getTimeout() { return timeout.get(); }
     public Identity getJoinstrIdentity() { return Identity.create(privateKey); }
 
```

Line 98: This catch block silently swallows all exceptions and has a meaningless null check (caught exceptions are never null).